### PR TITLE
how to configure repository

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

File: docs/how-to-configure-new-repository.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# How we configure repositories
+
+## Naming
+
+Most repositories have a name pattern like this:
+`open-telemetry/opentelemetry-foo`. The reason to include `opentelemetry` in the
+name of the repository is to simplify repositories distinction in forks. So when
+One will fork the repository - fork's name will still indicate that this
```
Comment (bogdandrutu) [inline docs/how-to-configure-new-repository.md]: Is One intentionally spelled with **O**?

File: docs/how-to-configure-new-repository.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Documents [Community Membership](../community-membership.md) and
+[CONTRIBUTING](../CONTRIBUTING.md) define how permissions are typically set up
+for the repository.
+
+1. Every repository has two teams associated with it. Typically they will be
+   named `foo-approvers` and `foo-maintainers`. `foo-maintainers` is a child of
+   `foo-approvers` as it always contains subset of people and defines larger
+   scope of privileges.
+2. Both teams has `Write` permissions for the repository.
+3. Root-level `CODEOWNERS` file on the repository should include superset of
```
Comment (bogdandrutu) [inline docs/how-to-configure-new-repository.md]: I prefer this under .github/ directory. What do you think?

File: docs/how-to-configure-new-repository.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+5. Some repositories may include more individuals outside of approvers and
+   maintainers teams with the `Write` permissions. Typically for issues tracking
+   and triage purpose.
+6. Some repositories may include more individuals with `Admin` permissions.
+   Typically to help set up repository, CI, web hooks or other administrative
+   work.
+
+![image](https://user-images.githubusercontent.com/9950081/57563719-d7b6b300-7355-11e9-9ebb-3c4f549336bc.png)
+
+## Policies
```
Comment (bogdandrutu) [inline docs/how-to-configure-new-repository.md]: We should also disable merge and rebase pull requests and allow only squash. What do you think?

File: docs/how-to-configure-new-repository.md (hunk 2 above)
Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]: hm, why hide it? It is a main source of information on who to ask about this repo. Is your concern a number of files in the root folder?

File: docs/how-to-configure-new-repository.md (hunk 3 above)
Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]: I'm fine with it. I like clean history. What do you think of the fact that authors of commits got lost after the squash? Seems unfair that contributors only stay in PRs history.

File: docs/how-to-configure-new-repository.md (hunk 2 above)
Comment (bogdandrutu) [reply docs/how-to-configure-new-repository.md]: Cleaner root directory. but this is minor, don't care where we put it, just want to enforce consistency.

File: docs/how-to-configure-new-repository.md (hunk 3 above)
Comment (bogdandrutu) [reply docs/how-to-configure-new-repository.md]: Maybe, but I also don't like to see commits like "fix review comments", "fix style", "fix tests". Very unclear what they do there.

File: docs/how-to-configure-new-repository.md (hunk 2 above)
Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]: Let's keep it in the root for now then.

File: docs/how-to-configure-new-repository.md (hunk 3 above)
Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]: I've added it as a best practice into the document

**Comment by SergeyKanzhelev** [comment]
I'm merging, but happy to accommodate more feedback.

**Pull request merged**